src/swift/README.md

```diff
@@ -0,0 +1,10 @@
+# APIView for Swift
+
+SwiftAPIView is a plugin that converts `.swift` source code to an APIView-compatible token file.
+
```

Comment:
Can we please list down any required tools or packages that needs to be installed to run this swift code.

Reply:
Done.